libraries/etcd_common_properties.rb

```diff
@@ -16,8 +16,9 @@ def self.included(base)
         #####################
         # set default service name like etcd.service
         property :default_service_name, kind_of: [TrueClass, FalseClass], default: false
+        property :etcd_bin_prefix, String, default: '/usr/local/bin', desired_state: false
```

Comment:
Default used to be `/usr/bin`. But from what I understand that is not what `/usr/bin` is for (https://unix.stackexchange.com/a/259238). `/usr/local/bin` seems more appropriate.